colorable_windows.go

```diff
@@ -13,7 +13,7 @@ import (
 	"syscall"
 	"unsafe"
 
-	"github.com/mattn/go-isatty"
+	isatty "github.com/mattn/go-isatty"
```

Comment:
Please revert this line.

Reply:
I've done that now!  Sorry that is `goimports` which I use on a save-hook